ansible/roles/slurm_node/tasks/sssd_enumerate.yml

```diff
@@ -0,0 +1,7 @@
+- name: Enable the enumerate option in sssd.conf
+  ansible.builtin.template:
+    src: 01_slurm.conf.j2
+    dest: /etc/sssd/conf.d/01_slurm.conf
+    owner: root
+    group: root
+    mode: 0644
```

Comment:
It would be best to also add a notify + handler to restart the sssd service on config change.

Reply:
Added now.